index.src.html

```diff
@@ -1380,6 +1380,28 @@ <h4 id="body-extraction" algorithm>
           6.  Set |Content-Type| to
               `application/x-www-form-urlencoded;charset=UTF-8`.
 
+          Otherwise, |data| is a generic object, so:
+
+          1.  If |data| has a property named like the |credential|'s
+              {{PasswordCredential/idName}} or
+              {{PasswordCredential/passwordName}}, <a>throw</a> a `TypeError`
+              exception.
```

Comment:
If I understand correctly, getters ar OK. Setters, which would take the password value as an argument, have potential for damage.